api/src/test/java/org/openmrs/module/fhir2/providers/r3/AllergyIntoleranceFhirR3ResourceProviderTest.java

```diff
@@ -323,4 +326,20 @@ public void searchForAllergies_shouldReturnMatchingBundleOfAllergiesByStatus() {
 		assertThat(((org.hl7.fhir.r4.model.AllergyIntolerance) resultList.iterator().next()).getId(), equalTo(ALLERGY_UUID));
 	}
 	
+	@Test
```

Comment:
@Akayeshmantha Can we add the update tests as well?

Reply:
@ibacher  sure I will do it